commands/login_test.go

```diff
@@ -3,6 +3,8 @@ package commands
 import (
 	"bufio"
 	"bytes"
+	ionoscloud "github.com/ionos-cloud/sdk-go/v5"
+	"os"
 	"testing"
 
 	"github.com/ionos-cloud/ionosctl/pkg/config"
```

Comment:
In Go, it is recommended to separate remote imports like `ionoscloud "github.com/ionos-cloud/sdk-go/v5"` for the other ones.

Reply:
done